tests/testthat/test-verb-select.R

```diff
@@ -248,6 +248,7 @@ test_that("mutate collapses over nested select", {
 
 test_that("output is styled", {
   local_reproducible_output(crayon = TRUE)
+  withr::local_options(dbplyr_highlight = cli::combine_ansi_styles("blue"))
```

Comment:
Why do we need this now?

Reply:
It is needed in case you use the `dbplyr_highlight` option and set it to something different than blue 😉